schema/me/append_conversation_thread.js

```diff
@@ -2,18 +2,30 @@ import impulse from "lib/loaders/impulse"
 import gravity from "lib/loaders/gravity"
 import { GraphQLString, GraphQLNonNull, GraphQLObjectType } from "graphql"
 const { IMPULSE_APPLICATION_ID } = process.env
-import { mutationWithClientMutationId } from "graphql-relay"
+import { mutationWithClientMutationId, cursorForObjectInConnection } from "graphql-relay"
+import { ConversationType, MessageEdge } from "./conversation"
 
-const MessagePayloadType = new GraphQLObjectType({
-  name: "MessagePayloadType",
-  description: "Expected payload from a successful message post",
+const getImpulseToken = gravityToken => {
+  return gravity.with(gravityToken, { method: "POST" })("me/token", {
+    client_application_id: IMPULSE_APPLICATION_ID,
+  })
+}
+
+const AppendConversationThreadMutationPayload = new GraphQLObjectType({
+  name: "AppendConversationThreadMutationPayload",
   fields: {
-    id: {
-      description: "Impulse id.",
-      type: new GraphQLNonNull(GraphQLString),
+    conversation: {
+      type: ConversationType,
+      resolve: ({ conversation }) => conversation,
```

Comment:
This one needs to grab the `conversation` from the `payload` object it receives.